podpac/coordinates.py

```diff
@@ -7,7 +7,8 @@
 
 from podpac.core.coordinates import Coordinates
 from podpac.core.coordinates import crange, clinspace
-from podpac.core.coordinates import Coordinates1d, ArrayCoordinates1d, UniformCoordinates1d, StackedCoordinates
+from podpac.core.coordinates import Coordinates1d, ArrayCoordinates1d, UniformCoordinates1d
+from podpac.core.coordinates import StackedCoordinates, DependentCoordinates
```

Comment:
Did you mean to include `RotatedCoordinates` here?

Reply:
Yes df6f1c0